src/test/functional/tests/addUser.spec.ts

```diff
@@ -0,0 +1,27 @@
+import { expect, test } from '@playwright/test';
+
+import { adminUserTestData } from '../data';
+import { LoginPage } from '../pages/LoginPage';
+import { PimPage } from '../pages/PimPage';
+
+let loginPage: LoginPage, pimPage: PimPage;
+
+test.describe('Adding a new employee', () => {
+  test.beforeEach(async ({ page }) => {
+    loginPage = new LoginPage(page);
+    pimPage = new PimPage(page);
+    await loginPage.initialize();
+    await loginPage.loginUser(adminUserTestData.userName, adminUserTestData.password);
+    await loginPage.navigateToSubPage('PIM');
+    await expect(page.getByRole('heading', { name: 'Employee Information' })).toBeVisible();
+  });
+
+  test.afterEach(async () => {
+    await loginPage.close();
+  });
+
+  test('Add new employee user with account  & fill new employee personal details', async () => {
```

Comment:
Well - I have a few suggestions regarding this test.
1. This test name isn't bad, but I think could be a little bit better. The test name should describe what should or should not have happened in test, not  what test will do in a process. Please consider renaming the test for something like "Should create a new user account with personal information" or "Data of newly created account and personal information should match provided data" (please next comment).